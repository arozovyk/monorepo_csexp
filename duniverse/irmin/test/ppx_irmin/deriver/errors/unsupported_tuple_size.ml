type t = unit * unit * unit * unit [@@deriving irmin]
