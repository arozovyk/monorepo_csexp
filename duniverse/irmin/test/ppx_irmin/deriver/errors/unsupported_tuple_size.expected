File "unsupported_tuple_size.ml", line 1, characters 0-53:
Error: ppx_irmin: tuple types must have 2 or 3 components. Found 4.
